### ISSUE TITLE
no attribute 'WIN_HAMMING' in Taps config for Polyphase Channelizer

### ISSUE THREAD

**Issue by crblackw** [argilo/gr-elster#7]
I'm receiving the following error with GRC 3.9.2.0 on Ubuntu 20.04 in the Polyphase Channelizer block of the elster_rx_nogui.grc adn elster_rx_multi.grc flows. Has any one else observed this error?

> Param - Taps(taps):
> 	Value "firdes.low_pass(1, samp_rate, 175000, 50000, firdes.WIN_HAMMING, 6.76)" cannot be evaluated:
> 	type object 'gnuradio.filter.filter_python.firdes' has no attribute 'WIN_HAMMING'

**Comment by crblackw** [comment]
Seems to have resolved itself. I have changed the taps entry from "firdes.low_pass(1, samp_rate, 175000, 50000, firdes.WIN_HAMMING, 6.76)" to "firdes.low_pass(1, samp_rate, 175000, 50000, 0, 6.76)". Ran it. Then changed it back to the original config referencing WIN_HAMMING and the error has gone away. I'm not that experienced with GNU Radio so I don't think I can provide much more context then this. Closing the issue.
